idea_town/frontend/static-src/app/lib/web-channel.js

```diff
@@ -23,9 +23,9 @@ class WebChannel {
       this.channelId = newChannelId;
     }
 
-    const message = evt.detail;
+    const message = evt.data;
```

Comment:
Getting this error now [from a few lines above](https://github.com/meandavejustice/idea-town/blob/install-flow-happy-path/idea_town/frontend/static-src/app/lib/web-channel.js#L21): `TypeError: evt.detail is null`

Not sure if the culprit is around here, though.